index.js

```diff
@@ -126,17 +129,22 @@ module.exports = function (options) {
                 }
 
                 if (options.minifier) {
-                    row.push(colors.bold(getSizeToDisplay(totalMinifiedSize, 'maxTotalMinifiedSize', '*')));    
+                    row.push(colors.bold(getSizeToDisplay(totalMinifiedSize, 'maxTotalMinifiedSize', '*')));
 
                     if (options.gzip) {
-                        row.push(colors.bold(getSizeToDisplay(totalMinifiedGzippedSize, 'maxTotalMinifiedGzippedSize', '*')));                        
+                        row.push(colors.bold(getSizeToDisplay(totalMinifiedGzippedSize, 'maxTotalMinifiedGzippedSize', '*')));
                     }
                 }
 
                 table.push(row);
             }
 
             console.log(table.toString());
+            if (options.fail) {
+                if(fail) {
```

Comment:
Could you please put those 2 conditions in 1 ?

    if (options.fail && fail) {